pkg/asyncexec/stream.go

```diff
@@ -0,0 +1,60 @@
+package asyncexec
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// StreamCommand executes a given command in a context directory and streams
+// the output to the according stdin/stdout.
+func StreamCommand(workDir string, command string, args []string) error {
+	cmd := exec.Command(command, args...)
+	if workDir != "" {
+		cmd.Dir = workDir
+	}
+
+	acmd := New(cmd, 256)
+	err := acmd.Run(true)
+	if err != nil {
+		return err
+	}
+
+	// done is used to wait for stream readers to finish before exiting the function.
+	done := make(chan struct{})
+
+	go func() {
+		for b := range acmd.StdoutStream() {
+			_, err := os.Stdout.Write(b)
+			if err != nil {
+				// should never happen, just panic.
+				panic("failed to write to Stdout")
```

Comment:
nit: Wrap with err message, for debugging

Reply:
++